client/src/layout/Mention.js

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect } from "react";
+import {
+  Typography,
+  Card,
+  CardContent,
+  CardMedia,
+  Box,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSmile } from "@fortawesome/free-regular-svg-icons";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.common.white,
+    display: "flex",
+    alignItems: "center",
+    marginRight: theme.spacing(3),
+    marginTop: theme.spacing(3),
+    width: "80%",
+    height: "10em",
+    borderRadius: theme.shape.borderRadius * 2,
+    boxShadow: "0px 0px 2px 0px rgba(0,0,0,0.3)",
+    "&:hover": {
+      boxShadow: "0px 0px 4px 1px rgba(0,0,0,0.3)",
+    },
+  },
+  media: {
+    minHeight: "8em",
+    minWidth: "8em",
+    marginLeft: "1em",
+    border: "1px solid black",
+  },
+  content: {
+    padding: theme.spacing(3),
+    display: "flex",
+    flexDirection: "column",
+    textAlign: "left",
+  },
+  subtitle: {
+    color: "#d2d2d2",
+    fontWeight: 400,
+    fontSize: 15,
+  },
+  text: {
+    color: theme.palette.text.secondary,
+  },
+  customBox: {
+    display: "-webkit-box",
+    boxOrient: "vertical",
+    lineClamp: 2,
+    wordBreak: "break-all",
+    overflow: "hidden",
+  },
+  icon: {
+    alignSelf: "start",
+    padding: theme.spacing(2),
+    color: theme.palette.primary.main,
+  },
+}));
+
+const Mention = ({ mention }) => {
+  const classes = useStyles();
+  const keyword = "DolphinCorp";
```

Comment:
What's the use case of the keyword here? I imagine it would make more sense for the keyword to be stored in a parent component so that all mentions can get it

Reply:
Yeah, you are right. The keyword should be extracted from match.params of the browser. The keyword here is to check if the UI is working or not since there won't be default keyword value without input it.

Reply:
Actually, we don't need to pass keywords into the browser since all info has been grabbed and we just need to do a filter. we can just store the keyword in the global store and have a function to update it right?